### ISSUE TITLE
Client-side implementation

### ISSUE THREAD

**Issue by CzechHek** [SilkMC/silk-compose#2]
Hey, I was just wondering, would it possible to open Compose screen like a modded fabric screen. So that it would be like ESC menu, Inventory screen and similar?

I couldn't even set up this mod in an IDE because I got out of memory on two different computers. Weird.

**Comment by jakobkmar** [comment]
Yes, a client-side implementation is possible, it is just that I needed the server-side one first.

The client side implementation would either render via LWJGL directly, or could render to an java.awt or Swing surface. We have to see what works out to be the best.

For developing this mod, you need at least 4GB of ram allocated to Gradle.
I have this in my `gradle.properties`:

```properties
org.gradle.jvmargs=-Xmx4g -XX:MaxMetaspaceSize=1g
```

Please note that the current version of the mod is a bit behind the most recent releases of Compose and Minecraft, but I will be picking it up again soon.
